### DISCUSSION TITLE
FR: Add alpha channel support to color fields

### DISCUSSION THREAD

**Discussion by evanwarner** [craftcms/cms#8983] [Ideas]
Would love it if color fields were RGBA instead of just RGB. The color preview already responds fine to appended alpha channel information, but the field won't save, giving a "Color isn’t a valid hex color value" error.

**Comment by piotrpog** [comment]
I think we still need this. I recently wanted to add hex value with alpha channel to color palette of color field and could not save it, validation error showed up.
Not allowing alpha values with colors is pretty limiting, any possibility alpha values will be allowed in the future?
